### ISSUE TITLE
Replace Locally Stored data.json With Ajax Calls to Dropbox API

### ISSUE THREAD

**Issue by Radai** [Radai/dnd5tools#32]
Referencing: https://www.dropbox.com/sh/hiavsiegq28xd7u/AABcMGhKcr8CYgeKaHK1ZDzJa?dl=0

**Comment by tomaskroth** [comment]
Try to make it so that it falls back to local json if the Dropbox call fails. Most companies proxies block Dropbox, and it's awesome that this tool works on its on.

**Comment by Radai** [comment]
The issue is that storing the data locally is probably a no bueno for wizards so I don't want to have a copy of the data hosted with this system.

**Comment by tomaskroth** [comment]
Yeah, that's true. Well, in any case I can branch it and change that for my local use. Awesome job you're doing here.
